projects/ion/src/lib/core/types/spinner.ts

```diff
@@ -0,0 +1,11 @@
+import { IonSpinnerColor } from '../../spinner/spinner.component';
```

Comment:
Why not bring this type to this file too, instead of importing it?

Reply:
I did it to avoid possible import problems somewhere. Should I still do it?